benchmarks/torchbench_model.py

```diff
@@ -144,6 +144,18 @@
     "hf_T5_generate",
 }
 
+FORCE_AMP_FOR_FP16_BF16_MODELS = {
```

Comment:
Do you know how this is configured on PyTorch HUD? Maintaining a list like this feels prone to divergence.

Reply:
These lists are taken from the scripts in the PyTorch main repo. They are used for generating the PyTorch HUD results.

Reply:
I will leave a comment.

Reply:
Ty

Reply:
Could you share the link to the scripts in the PyTorch main repo that are used to generate the HUD result?